### PR TITLE
Update nftokenmint

### PR DESCRIPTION
**Pull request by DennisDawson**
A small but critical clarification to the tfTransferable setting.

### CODE REVIEW

**Comment by DennisDawson** [comment]
Not sure why the commit for the intro graphics was included in this PR, but this is not ready for merge. I'm going to go with it, though, to see if I can get a build of the draft docs.

File: content/references/protocol-reference/transactions/transaction-types/nftokenmint.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ Transactions of the NFTokenMint type support additional values in the [`Flags` f
 | `tfBurnable` | `0x00000001` | 1 | Allow the issuer (or an entity authorized by the issuer) to destroy the minted `NFToken`. (The `NFToken`'s owner can _always_ do so.) |
 | `tfOnlyXRP` | `0x00000002` | 2 | The minted `NFToken` can only be bought or sold for XRP. This can be desirable if the token has a transfer fee and the issuer does not want to receive fees in non-XRP currencies. |
 | `tfTrustLine` | `0x00000004` | 4 | **DEPRECATED** Automatically create [trust lines](trust-lines-and-issuing.html) from the issuer to hold transfer fees received from transferring the minted `NFToken`. The [fixRemoveNFTokenAutoTrustLine amendment][] makes it invalid to set this flag. |
-| `tfTransferable` | `0x00000008` | 8 | The minted `NFToken` can be transferred to others. If this flag is _not_ enabled, the token can still be transferred _from_ or _to_ the issuer. |
+| `tfTransferable` | `0x00000008` | 8 | The minted `NFToken` can be transferred to others. If this flag is _not_ enabled, the token can still be transferred _from_ or _to_ the issuer. The transfer to the issuer must be via a buy offer created by the issuer and not a sell offer from the folder of the `NFToken`. |
```
Comment (kennyzlei) [inline content/references/protocol-reference/transactions/transaction-types/nftokenmint.md]: typo: folder => holder

Comment (DennisDawson) [reply content/references/protocol-reference/transactions/transaction-types/nftokenmint.md]: That's a weird typo. Chalk it up to age. Thanks for catching that.

**Comment by DennisDawson** [comment]
These changes have been made under a separate PR.

**Pull request closed without merging**